Source/VCGeneration/Split.cs

```diff
@@ -69,6 +69,7 @@ void ObjectInvariant()
 
 
       private readonly List<Block> blocks;
+      public IEnumerable<AssertCmd> Asserts => blocks.SelectMany(block => block.cmds.OfType<AssertCmd>());
```

Comment:
This only seems to be used by the class its defined in and only once. If that's the case, consider moving this to a local variable.